cmd/temporalite/main.go

```diff
@@ -169,6 +185,19 @@ func buildCLI() *cli.App {
 					return err
 				}
 
+				baseConfig := &temporalconfig.Config{}
+				if c.IsSet(configFileFlag) {
+					b, err := os.ReadFile(c.String(configFileFlag))
+					if err != nil {
+						return err
+					}
+
+					err = yaml.Unmarshal(b, baseConfig)
+					if err != nil {
+						return err
+					}
+				}
```

Comment:
If we update the config argument to accept a directory (as temporal server does), then we can use the https://pkg.go.dev/go.temporal.io/server/common/config#LoadConfig function to read in its value.

Reply:
It looks like that load func supports merging multiple config files, so we could set the environment to `temporalite` and the availability zone to an empty string so that in a directory structured like

```
config
- base.yaml
- temporalite.yaml
```

`temporalite.yaml` would inherit from `base.yaml` if it exists.